tests/phpunit/Config/ApiTest.php

```diff
@@ -102,18 +112,20 @@ public function testCreateAuthQueryDeprecated(): void
         $apiConfig = [
             'baseUrl' => 'http://example.com',
             'authentication' => [
-                'type' => 'url.query'
+                'type' => 'url.query',
             ],
             'query' => [
                 'param' => [
-                    'attr' => 'key'
-                ]
-            ]
+                    'attr' => 'key',
+                ],
+            ],
         ];
         $api = new Api(new NullLogger(), $apiConfig, $attributes, []);
         $request = new Request('GET', 'http://example.com?foo=bar');
         $restClientMock = $this->getClientMock($request);
-        /** @var RestClient $restClientMock */
+        /**
+ * @var RestClient $restClientMock
```

Comment:
pořád tu ještě některý zbývají - ničemu to nevadí, ale bylo by fajn je všechny mít zarovnaný stejně

Reply:
opravim v dalsom PR, snazil som sa najst vsetky vyskyty cez GREP ale asi som to napisal blbo